app/actions/remote/channel.ts

```diff
@@ -1145,7 +1145,7 @@ export async function switchToLastChannel(serverUrl: string, teamId?: string) {
     }
 }
 
-export async function searchChannels(serverUrl: string, term: string) {
+export async function searchChannels(serverUrl: string, term: string, isSearch = false) {
```

Comment:
@larkox, this might make more sense as a variable name to this function? 
```suggestion
export async function searchChannels(serverUrl: string, term: string, forSearch = false) {
```